app/api/v1/endpoints/auth.py

```
@@ -485,54 +485,134 @@ async def refresh_token(
+            except Exception as db_error:
+                logger.warning("Database session cleanup failed - non-critical", error=str(db_error))
+                try:
+                    await db.rollback()
+                except:
+                    pass
+        
+        # ENTERPRISE: Additional cleanup - remove expired sessions (housekeeping)
+        try:
+            expired_result = await db.execute(
+                delete(UserSession).where(
+                    UserSession.expires_at < func.now()
```

Ruff (0.12.2)

Undefined name `func`

(F821)